roles/redpanda_broker/tasks/start-redpanda.yml

```diff
@@ -92,12 +92,32 @@
   tags:
     rp_tuner
 
+- name: Create file with RP_BOOTSTRAP_USER for systemd
+  ansible.builtin.copy:
+    dest: "/etc/redpanda.d/superuser.conf"
+    content: "RP_BOOTSTRAP_USER={{ sasl_bootstrap_admin_user }}:{{ sasl_bootstrap_admin_pw }}\n"
+    owner: "redpanda"
+    group: "redpanda"
+    mode: "0600"
+  when:
+    - not is_initialized
+    - enable_sasl is true
+
 # ensure that systemd starts up redpanda
 - name: Start redpanda
   ansible.builtin.systemd:
     name: redpanda
     state: started
 
+# we don't need this file after RP has started
+- name: Delete file with RP_BOOTSTRAP_USER
```

Comment:
I didn't know the user persisted after restarts if it was removed from the initial config, interesting. But, I also never use the bootstrap user outside of dev testing heh.

Reply:
Yep absolutely does.